cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1682,23 +1745,23 @@ subroutine stressC_T  (nx_block, ny_block , &
          indxTj       ! compressed index in j-direction
 
       real (kind=dbl_kind), dimension (nx_block,ny_block), intent(in) :: &
-         uvelE    , & ! x-component of velocity (m/s) at the E point
-         vvelE    , & ! y-component of velocity (m/s) at the E point
-         uvelN    , & ! x-component of velocity (m/s) at the N point
-         vvelN    , & ! y-component of velocity (m/s) at the N point
-         dxN      , & ! width of N-cell through the middle (m)
-         dyE      , & ! height of E-cell through the middle (m)
-         dxT      , & ! width of T-cell through the middle (m)
-         dyT      , & ! height of T-cell through the middle (m)
-         strength , & ! ice strength (N/m)
-         shearU   , & ! shearU
-         uarea    , & ! area of u cell
-         DminTarea    ! deltaminEVP*tarea
+         uvelE     , & ! x-component of velocity (m/s) at the E point
+         vvelE     , & ! y-component of velocity (m/s) at the E point
+         uvelN     , & ! x-component of velocity (m/s) at the N point
+         vvelN     , & ! y-component of velocity (m/s) at the N point
+         dxN       , & ! width of N-cell through the middle (m)
+         dyE       , & ! height of E-cell through the middle (m)
+         dxT       , & ! width of T-cell through the middle (m)
+         dyT       , & ! height of T-cell through the middle (m)
+         strength  , & ! ice strength (N/m)
+         shearU, & ! shearU local for this routine
```

Comment:
Thanks for cleaning up in various places.  Please make another pass through the diffs to check, e.g. here the column alignment is off.